### PR TITLE
stylized the shotguns

### PR DESCRIPTION
**Pull request by bleehu**
I put the range, miss chance and damage into a table. Double-check my transcription to make sure I didn't accidentally typo something. I did this when a bit sleepy.

### CODE REVIEW

File: CharacterCreation/04_03_SMGs_&_Shotguns.txt [hunk 1]
```diff
@@ -209,57 +267,90 @@ Auto Fire Rate: 	2
...
+| Damage  |  Miss Chance |  Out to  |
++---------+--------------+----------+
+|   140   |      3       |    10m   |   
++---------+--------------+----------+
+|   105   |      4       |    20m   |
++---------+--------------+----------+
+|   115   |5 (Slug only) |    50m   |
++---------+--------------+----------+
+
+	Slug shells do 120 damage for all range brackets
```
Comment (Turtlelord26) [inline CharacterCreation/04_03_SMGs_&_Shotguns.txt]: Deleted line 271 says slug damage is 115.

File: CharacterCreation/04_03_SMGs_&_Shotguns.txt [hunk 2]
```diff
@@ -283,36 +382,53 @@ Auto Fire Rate: 	4
...
+== Frontier Plas-12	$3253	==
+
++---------+--------------+----------+
+| Damage  |  Miss Chance |  Out to  |
++---------+--------------+----------+
+|   160   |      1       |    10m   |   
++---------+--------------+----------+
+|   140   |      3       |    20m   |
++---------+--------------+----------+
+|   110   |4 (Slug only) |    35m   |
```
Comment (Turtlelord26) [inline CharacterCreation/04_03_SMGs_&_Shotguns.txt]: This gun makes no mention of slugs in the deleted lines. Like one of the plasmas above, I suspect it was intended to be buck only.

File: CharacterCreation/04_03_SMGs_&_Shotguns.txt [hunk 3]
```diff
@@ -283,36 +382,53 @@ Auto Fire Rate: 	4
...
++---------+--------------+----------+
+
+	Damage dealt with this weapon is plasma damage. Slug shells do 120 damage
```
Comment (Turtlelord26) [inline CharacterCreation/04_03_SMGs_&_Shotguns.txt]: Second dubious reference to slugs.

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
This is good handling of formatting shotguns' different damage mechanics. I understand why you didn't do it for the Matthews 2287 (the one with only slugs... - must not scope creep), but I think the positives from consistency within weapon families outweigh the lesser need for that table for this particular shotgun. That is an opinion, not a style blocker.

Some possible number transcription errors noted.

File: CharacterCreation/04_03_SMGs_&_Shotguns.txt (hunk 1 above)
Comment (bleehu) [reply CharacterCreation/04_03_SMGs_&_Shotguns.txt]: very good catch. Good eye.

File: CharacterCreation/04_03_SMGs_&_Shotguns.txt (hunk 3 above)
Comment (bleehu) [reply CharacterCreation/04_03_SMGs_&_Shotguns.txt]: yanked it

**Review by Turtlelord26 — APPROVED** [review]

**Pull request merged**